packages/node_modules/@ciscospark/react-container-message-composer/src/container.js

```diff
@@ -130,18 +130,20 @@ export class MessageComposer extends Component {
     } = this.props;
 
     const participants = [];
-    conversation.get(`participants`).toJS().forEach((p) => {
-      if (user.get(`currentUser`).id !== p.id) {
-        participants.push({
-          display: p.displayName,
-          id: p.id,
-          avatarUrl: avatar.getIn([`items`, p.id])
-        });
-      }
-    });
+    if (conversation.get(`participants`)) {
```

Comment:
Might be better to do `conversation.has(` instead?

Reply:
Actually I think that passes as true every time as long as we've retrieved the conversation. `.has()` just checks if the key exists, so we should probably do `conversation.get(`participants`).first()` or `conversation.get(`participants`).count()`